app/src/main/AndroidManifest.xml

```diff
@@ -4,6 +4,7 @@
 
     <uses-permission android:name="android.permission.INTERNET" />
     <application
+        android:name=".SuperHeroApplication"
```

Comment:
This is required for the system to know how to locate the Application class you created.